src/app/components/form/input/FormInputPassword.vue

```diff
@@ -11,9 +11,20 @@
     <template v-if="isStrengthShown" #inputSuffix>
       <Progress :model-value="strength" class="my-2" />
     </template>
+    <template #clearButton>
+      <ButtonIcon
+        :aria-label="t('iconAltClose')"
+        class="hidden flex-shrink-0 px-2 peer-[:not([data-empty='true'])]:block peer-[&[aria-invalid=true]]:text-(--semantic-critic-text)"
+        @click="emit('input', '')"
+      >
+        <IVibetypeClose :aria-label="t('iconAltClose')" class="size-6" />
+      </ButtonIcon>
+    </template>
```

Comment:
Why define this `<template #clearButton>` in child components of `FormInput` which has it defined as default slot content?